src/token-list.json

```diff
@@ -302,34 +378,44 @@
     }
   },
   {
+    "chainId": 101,
+    "address": "7vfCXTUXx5WJV5JADk17DUJ4ksgau7utNKj4b963voxs",
     "symbol": "whETH",
     "name": "Ether (Wormhole)",
-    "address": "7vfCXTUXx5WJV5JADk17DUJ4ksgau7utNKj4b963voxs",
     "decimals": 8,
     "logoURI": "https://cdn.jsdelivr.net/gh/saber-hq/spl-token-icons@master/icons/101/2FPyTwcZLUg1MDrwsyoP4D6s1tM7hAkHYRjkNb5w6Pxk.png",
+    "tags": ["saber-market-eth", "wormhole-v2"],
     "extensions": {
       "coingeckoId": "ethereum"
-    },
-    "chainId": 101,
-    "tags": [
-      "saber-market-eth",
-      "wormhole-v2"
-    ]
+    }
   },
   {
-    "name": "Saber Wrapped FTT (Wormhole) (9 decimals)",
+    "chainId": 101,
     "address": "FTT9GrHBVHvDeUTgLU8FxVJouGqg9uiWGmmjETdm32Sx",
-    "decimals": 9,
     "symbol": "swFTT-9",
-    "chainId": 101,
+    "name": "Saber Wrapped FTT (Wormhole) (9 decimals)",
+    "decimals": 9,
     "logoURI": "https://cdn.jsdelivr.net/gh/saber-hq/spl-token-icons@master/icons/101/AGFEad2et2ZJif9jaGpdMixQqvW5i81aBdvKe7PHNfz3.png",
-    "tags": [
-      "saber-market-ftt",
-      "wormhole-v2",
-      "saber-decimal-wrapped"
-    ],
+    "tags": ["saber-market-ftt", "wormhole-v2", "saber-decimal-wrapped"],
     "extensions": {
       "coingeckoId": "ftx-token"
     }
+  },
+  {
+    "chainId": 101,
+    "address": "DYDWu4hE4MN3aH897xQ3sRTs5EAjJDmQsKLNhbpUiKun",
+    "symbol": "pBTC",
+    "name": "pBTC (Parrot BTC)",
+    "decimals": 6,
+    "logoURI": "https://raw.githubusercontent.com/solana-labs/token-list/main/assets/mainnet/DYDWu4hE4MN3aH897xQ3sRTs5EAjJDmQsKLNhbpUiKun/logo.svg",
+    "tags": ["stablecoin"],
+    "extensions": {
+      "website": "https://parrot.fi",
+      "twitter": "https://twitter.com/gopartyparrot",
```

Comment:
We can cull out any of the socials. We only need serum market info, website, and coingeckoID.

Reply:
Thanks for the reminder. Going to leave these and reorganize that change in-memory.